frontend/src/host_orchestrator/orchestrator/controller.go

```diff
@@ -176,11 +177,7 @@ func (h *createCVDHandler) Handle(r *http.Request) (interface{}, error) {
 		http.DefaultClient, h.Config.AndroidBuildServiceURL, buildAPIOpts)
 	artifactsFetcher := newBuildAPIArtifactsFetcher(buildAPI)
 	cvdBundleFetcher := newFetchCVDCommandArtifactsFetcher(exec.CommandContext, creds)
-	cvdStartTimeout := 3 * time.Minute
-	if req.EnvConfig != nil {
-		// Use a lengthier timeout when using canonical configs as this operation downloads artifacts as well.
-		cvdStartTimeout = 7 * time.Minute
-	}
+	cvdStartTimeout := time.Duration(h.Config.CVDCreationTimeout) * time.Minute
```

Comment:
Do we have any explanation about the unit of `cvdStartTimeout`?

Reply:
Changed h.Config.CVDCreationTimeout from int to time.Duration so it just represents time duration and does not need to care about unit.